_posts/2021-06-03-opensearch-release-candidate-announcement.markdown

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title:  "OpenSearch Release Candidate (RC1) is now available"
+authors: 
+  - andhopp
+date:   2021-06-07
+categories: update
+---
+
+Today we are excited to announce the Release Candidate (RC1) for version 1.0.0 of both OpenSearch (derived from Elasticsearch 7.10.2) and OpenSearch Dashboards (derived from Kibana 7.10.2). The Release Candidate includes downloadable [artifacts](https://opensearch.org/downloads.html) (Linux tars and Docker images) as well as a number of OpenSearch plugins (listed below). 
+
+**So what does a release candidate mean exactly?**
+
+This Release Candidate is a version of the project that is feature complete and passing automated testing with the intent to validate expected functionality before moving to a GA launch. The goal with this Release Candidate is to share the current release with the community and solicit a final round of testing and feedback before the GA release. Between now and the 1.0.0 release of OpenSearch and OpenSearch Dashboards, we expect no further changes apart from critical bug fixes and the release of additional artifacts (see below). 
```

Comment:
you need to define what acronym 'GA' means

Reply:
Added, thanks!